### ISSUE TITLE
An error occurred while exporting to Open in image editor: Parameter is not valid.

### ISSUE THREAD

**Issue by badabing2005** [greenshot/greenshot#307]
Using version 1.3.194 I encountered the following error when I attempted to open an png file (any png file behaved the same way, even previously properly opened ones)

```
Software version: 1.3.194 - 54038a3cbe (64 bit)
.NET runtime version: 4.0.30319.42000+
Time: 2021-04-05 10:02:15 -04:00
OS: Windows 10 2009 x64 build 19042
GDI object count: 149
User object count: 94

Exception: System.ArgumentException
Message: Parameter is not valid.

Stack:
   at System.Drawing.Image.FromStream(Stream stream, Boolean useEmbeddedColorManagement, Boolean validateImageData)
   at Greenshot.Base.Core.ImageHelper.<.cctor>g__DefaultConverter|3_2(Stream stream, String s) in D:\a\1\s\src\Greenshot.Base\Core\ImageHelper.cs:line 85
   at Greenshot.Base.Core.ImageHelper.FromStream(Stream stream, String extension) in D:\a\1\s\src\Greenshot.Base\Core\ImageHelper.cs:line 1791
   at Greenshot.Base.Core.ClipboardHelper.ContainsImage(IDataObject dataObject) in D:\a\1\s\src\Greenshot.Base\Core\ClipboardHelper.cs:line 321
   at Greenshot.Forms.ImageEditorForm.UpdateClipboardSurfaceDependencies() in D:\a\1\s\src\Greenshot\Forms\ImageEditorForm.cs:line 1168
   at Greenshot.Forms.ImageEditorForm.ImageEditorFormActivated(Object sender, EventArgs e) in D:\a\1\s\src\Greenshot\Forms\ImageEditorForm.cs:line 859
   at System.Windows.Forms.Form.OnActivated(EventArgs e)
   at System.Windows.Forms.Form.set_Active(Boolean value)
   at System.Windows.Forms.Form.WmActivate(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

Configuration dump:
```
The screen was blank, and the status bar was displaying the following
![image](https://user-images.githubusercontent.com/22603312/113582822-d734fe80-95f6-11eb-8d7a-c18504c7f8b3.png)

When I quit Greenshot and started it again, everything started behaving normal

![image](https://user-images.githubusercontent.com/22603312/113582903-f03daf80-95f6-11eb-946c-fb526aeee554.png)

All good now, but thought I'd report it,

Thanks

**Comment by Lakritzator** [comment]
Thanks!
We unfortunately don't have any telemetry, would be too expensive for free software, so we don't get errors unless they are reported...

I really can't think of a reason for this happening, as restarting worked it confirms my suspicion that "something" was wrong, but I will keep an eye out.

**Comment by badabing2005** [comment]
Hi,
Not sure if it is related to the above or not, I experienced a similar problem again today, after that I couldn't bring up the app from taskbar or tray icon, it would immediately generate this error.
In fact I couldn't even quit, because closing the window would generate another error.

Hope this helps.
Thanks
```
Software version: 1.3.194 - 54038a3cbe (64 bit)
.NET runtime version: 4.0.30319.42000+
Time: 2021-04-09 06:08:50 -04:00
OS: Windows 10 2009 x64 build 19042
GDI object count: 211
User object count: 122

Exception: System.Runtime.InteropServices.COMException
Message: Invalid FORMATETC structure (Exception from HRESULT: 0x80040064 (DV_E_FORMATETC))

ErrorCode: 0x80040064

Stack:
   at System.Runtime.InteropServices.ComTypes.IDataObject.GetData(FORMATETC& format, STGMEDIUM& medium)
   at System.Windows.Forms.DataObject.System.Runtime.InteropServices.ComTypes.IDataObject.GetData(FORMATETC& formatetc, STGMEDIUM& medium)
   at Greenshot.Base.Core.FileDescriptorReader.GetFileContents(IDataObject dataObject, Int32 index) in D:\a\1\s\src\Greenshot.Base\Core\FileDescriptorReader.cs:line 156
   at Greenshot.Base.Core.ClipboardHelper.ContainsImage(IDataObject dataObject) in D:\a\1\s\src\Greenshot.Base\Core\ClipboardHelper.cs:line 314
   at Greenshot.Forms.ImageEditorForm.UpdateClipboardSurfaceDependencies() in D:\a\1\s\src\Greenshot\Forms\ImageEditorForm.cs:line 1168
   at Greenshot.Forms.ImageEditorForm.ImageEditorFormActivated(Object sender, EventArgs e) in D:\a\1\s\src\Greenshot\Forms\ImageEditorForm.cs:line 859
   at System.Windows.Forms.Form.OnActivated(EventArgs e)
   at System.Windows.Forms.Form.set_Active(Boolean value)
   at System.Windows.Forms.Form.WmActivate(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

Configuration dump:
```

**Comment by badabing2005** [comment]
again, not sure if it is related to the above, this I have never experienced before,
I have a plugin configured to open the current image in explorer
![image](https://user-images.githubusercontent.com/22603312/114429144-074a4780-9b8b-11eb-91c5-0d55315b10f3.png)
This has been working for the longest time, and even had worked with this version of Greenshot, 1.3.194
But somehow today, when I invoked the plugin, the file was not revealed and I got this in the status bar

![image](https://user-images.githubusercontent.com/22603312/114429323-41b3e480-9b8b-11eb-8b2e-adcea101f4c6.png)

Double checked and the file does exist, even saved again and still same issue.
Quit / restart greenshot and it starts working again.

**Comment by Lakritzator** [comment]
I found that while we change the clipboard contents, there was a bug in the logic for that, as this was called on many locations it caused all kinds of issues. With the most recent changes, this should be a thing of the past. The fix will be in 1.3.197 or later, which I expect to be available later today.

**Comment by badabing2005** [comment]
Thanks,

Where is 1.3.197 release posted?
All I see is 1.3.194.

![image](https://user-images.githubusercontent.com/22603312/119257564-ca16a380-bb93-11eb-893d-bb175fff0411.png)

Thanks
